back/app/views/application/_logo_medium_top.mjml

```diff
@@ -1,9 +1,5 @@
 <mj-section padding="20px 25px 0">
-  <mj-column>
-    <mj-raw>
-      <%= link_to home_url do %>
-        <%= image_tag logo_url, alt: 'Organization logo' %>
-      <% end %>
-    </mj-raw>
+  <mj-column css-class="col">
```

Comment:
```suggestion
  <mj-column>
```
Do you ever use this `col` class? If not, you could delete it.

Reply:
Oops, my bad.